Tools/DumpRenderTree/DumpRenderTree.xcodeproj/project.pbxproj

```diff
@@ -1180,13 +1180,15 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = BCB281F00CFA713D007E533E /* DumpRenderTree.xcconfig */;
 			buildSettings = {
+				PRODUCT_BUNDLE_IDENTIFIER = com.apple.WebKit.DumpRenderTree;
```

Comment:
Could you make this change in `Tools/DumpRenderTree/mac/Configurations/DumpRenderTree.xcconfig` instead?